keyboards/redragon/k552/v2/keymaps/via/k552_via.json

```diff
@@ -0,0 +1,137 @@
+{
+    "name": "Redragon K552 ANSI",
```

Comment:
```suggestion
    "name": "Redragon K552 ISO",
```

Reply:
Done and also changed the name for the USB device so it's consistent/matching.